py-polars/polars/io/database.py

```diff
@@ -21,6 +21,7 @@ def read_database(
     partition_num: int | None = None,
     protocol: str | None = None,
     engine: DbReadEngine = "connectorx",
+    adbc_db_kwargs: dict[str, str] | None = None,
```

Comment:
Can you change `adbc_db_kwargs` to `adbc_options`, since they don't present at the top level as kwargs, and the "db" is somewhat redundant given that it's clearly a database method :)

Reply:
Agreed and done!

Reply:
> Agreed and done!

Ahh, I think you may have updated before I added the "Changing my mind" comment; can you update it to just `db_options` instead?

Reply:
Hehe, I agree.

Reply:
Done!